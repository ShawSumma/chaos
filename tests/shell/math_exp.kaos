3 + 5
145 + 24.8
0.32 + 0.42
(3 + 5) * 2
