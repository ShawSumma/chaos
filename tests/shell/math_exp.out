[0;90m###[0m 8
[0;90m###[0m 169.800003
[0;90m###[0m 0.740000
[0;90m###[0m 16
[0;90m###[0m 